.storybook/pages/FeedbackOverview/FeedbackOverview.stories.ts

```diff
@@ -10,10 +10,17 @@ export default {
 
 type Args = ComponentProps<typeof FeedbackOverview>;
 
-export const Overview: StoryObj<Args> = {};
+export const Overview: StoryObj<Args> = {
+  parameters: {
+    chromatic: { viewports: [414, 768, 1366] },
```

Comment:
Thanks for adding these! I'm wondering though if we need to switch the page to the main content on one or more of these so we actually get the main content of the page and not just the table of contents.

Reply:
Agree that the smaller viewports would have more value if the page could be switched to the main content, but not sure how to achieve this without using `play` functions, and since the nav to content is a css transition, the results are flaky with Chromatic